docs/DeveloperGuide.adoc

```diff
@@ -820,24 +820,30 @@ image::UndoAndRedo/RunCommandActivityDiagrams.png[]
 // tag::Filtering transactions[]
 === Filtering transactions
 
-The *List* command allows the user to bring up a list of `**Transactions**`, and filter it by month or by its tags.
-This feature is implemented by using a `**Predicate<Transaction>**` to filter the `**Transactions**` by month or tags.
+The *List* command allows the user to bring up a list of `**Transactions**`, and filter it by month.
+This feature is implemented by using a `**Predicate<Transaction>**` to filter the `**Transactions**` by month.
 
 ==== Implementation
 `**ListCommand**` is instantiated by `ListCommandParser#parse(String args)`,
 which attempts to parse the various parameters supplied in `args` and return a `**ListCommand**` object.
 
+The sequence diagram below shows how the execution of the **list** is like.
+
+image::list/ListSequenceDiagram.png[]
```

Comment:
updateFilteredTransactionList(...) is called by ListCommand and not Model itself. (Applies to the setCurrentMonthYear() & updateFilteredTransactionListToCurrentMonth())

Reply:
changed Model -> Model: xxxMethod() to ListCommand -> Model: xxxMethod() in the plantuml.